packages/lib/gpcircuits/src/circuitParameters.json

```diff
@@ -8,9 +8,11 @@
       "maxLists": 0,
       "maxListElements": 0,
       "maxTuples": 0,
-      "tupleArity": 0
+      "tupleArity": 0,
+      "includeOwnerV3": true,
+      "includeOwnerV4": true
     },
-    6607
+    8082
```

Comment:
Oof, this is a significant cost.
We should probably have most of our default circuits have only one type of owner.